exasol_transformers_extension/deployment/language_container_deployer_cli.py

```diff
@@ -26,6 +26,8 @@
               default=lambda: os.environ.get(
                   utils.DB_PASSWORD_ENVIRONMENT_VARIABLE, ""))
 @click.option('--language-alias', type=str, default="PYTHON3_TE")
+@click.option('--ssl_cert_path', type=str, default="")
+@click.option('--use_ssl_cert_validation', type=bool, default=True)
```

Comment:
```suggestion
@click.option('--use-ssl-cert-validation/--no-use-ssl-cert-validation', type=bool, default=True)
```

https://click.palletsprojects.com/en/8.1.x/options/#boolean-flags